content/en/community/static/community-values.md

```diff
@@ -0,0 +1,28 @@
+<!-- Do not edit this file directly. Get the latest from
```

Comment:
This document makes extensive use of passive voice and would benefit from rewriting. This would be a great first issue for a contributor with strong English writing skills. (See https://github.com/kubernetes/community/issues/5408)

Like the comment says, make changes upstream.

Reply:
I'll update once https://github.com/kubernetes/community/pull/5411 is merged. 👍